doc/dynamic-environments/quickstart.mkd

```diff
@@ -3,22 +3,21 @@
 This intent of this document is to serve as a basic guide for getting started with r10k and a fresh Puppet installation, including the following:
 
 * Installing Puppet and its dependencies
```

Comment:
I'm fine with specifying "Puppet 4 and greater", but it's worthwhile saying that this won't work at all for Puppet 3.x.